src/test/java/seedu/guest/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -27,7 +27,7 @@ public void setUp() {
 
     @Test
     public void execute_newPerson_success() {
```

Comment:
Should this method name be newGuest instead of newPerson?

Reply:
Maybe it is irrelevant for now for your PR as it seems you were refactoring to add the number of guests as a field!

Reply:
Will be done by Bryan under refactoring tests.